client/src/App.js

```diff
@@ -1,10 +1,12 @@
 import Page from "./components/pages/Page.js";
-import "./App.css";
+import GlobalStyle from "./GlobalStyle";
 import React from "react";
+import Input from "../src/components/Input/Input";
 
 function App() {
   return (
     <div className="App">
```

Comment:
You don't need the className anymore when using styled-components and you already deleted your `App.js`

Reply:
What do you mean by "already deleted your app.js"?

Reply:
In this case, I do need the App class, because I call it in the Global Style. Or am I wrong with this?

Reply:
I meant `App.css` sorry for that. And I guess you can do that, however I am not quite sure whether that is best practice or not.